.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
   "prettier.singleQuote": true,
   "prettier.printWidth": 100,
   "prettier.tabWidth": 2,
-  "prettier.trailingComma": "all",
```

Comment:
Why did you remove the trailing commas?

Reply:
@xylocopaCaerulea I removed it because it was adding unnecessary commas.